apps/judicial-system/web/src/routes/Shared/CaseTableGroups/CaseTableGroups.tsx

```diff
@@ -49,6 +54,16 @@ const CaseTableGroups: FC = () => {
               ))}
             </CasesDashboardLayout>
           ))}
+          {user && isProsecutionUser(user) ? (
```

Comment:
The `user` parameter is optional in `isProsecutionUser` so you should be able to do 

```suggestion
          {isProsecutionUser(user) ? (
```

right?

Reply:
It is required in `CreateCaseButton`

Reply:
Ahh I see. Nevermind then :)